[debug] [2023-11-20T12:35:31.313Z] ----------------------------------------------------------------------
[debug] [2023-11-20T12:35:31.315Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2023-11-20T12:35:31.315Z] CLI Version:   12.7.0
[debug] [2023-11-20T12:35:31.316Z] Platform:      win32
[debug] [2023-11-20T12:35:31.316Z] Node Version:  v18.16.0
[debug] [2023-11-20T12:35:31.316Z] Time:          Mon Nov 20 2023 18:05:31 GMT+0530 (India Standard Time)
[debug] [2023-11-20T12:35:31.317Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-11-20T12:35:31.320Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2023-11-20T12:35:31.463Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-11-20T12:35:31.463Z] > authorizing via signed-in user (broaddcast@gmail.com)
[debug] [2023-11-20T12:35:31.463Z] [iam] checking project driftly-1 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2023-11-20T12:35:31.464Z] > refreshing access token with scopes: []
[debug] [2023-11-20T12:35:31.465Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2023-11-20T12:35:31.465Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2023-11-20T12:35:31.470Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: getaddrinfo ENOTFOUND firebase-public.firebaseio.com
[warn] !  Unable to fetch the CLI MOTD and remote config. 
[debug] [2023-11-20T12:35:31.472Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2023-11-20T12:35:31.472Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------547319028564971783254525","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2023-11-20T12:35:31.473Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2023-11-20T12:35:31.473Z] > refreshing access token with scopes: []
[debug] [2023-11-20T12:35:31.473Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2023-11-20T12:35:31.473Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2023-11-20T12:35:31.482Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------588317263542104340193373","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2023-11-20T12:35:31.482Z] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2023-11-20T12:35:31.486Z] FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
    at invalidCredentialError (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\auth.js:144:12)
    at refreshTokens (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\auth.js:508:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Client.getAccessToken (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:149:22)
    at async Client.addAuthHeader (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:140:21)
    at async Client.request (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:96:34)
    at async getFirebaseProject (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\management\projects.js:295:21)
    at async getDefaultHostingSite (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\getDefaultHostingSite.js:10:21)
    at async requireHostingSite (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\requireHostingSite.js:9:18)
    at async Object.fn (C:\Users\BROADDCAST\AppData\Roaming\npm\node_modules\firebase-tools\lib\commands\deploy.js:74:9)
[error] 
[error] Error: Failed to get Firebase project driftly-1. Please make sure the project exists and your account has permission to access it.
